src/lib/svg_text_utils.js

```diff
@@ -173,70 +173,149 @@ function cleanEscapesForTex(s) {
         .replace(GT_MATCH, '\\gt ');
 }
 
+var inlineMath = [['$', '$'], ['\\(', '\\)']];
+
 function texToSVG(_texString, _config, _callback) {
+    var MathJaxVersion = parseInt(
+        (MathJax.version || '').split('.')[0]
+    ) || -1;
+
+    if(!MathJaxVersion) {
+        Lib.warn('No MathJax version:', MathJax.version);
+    }
+
     var originalRenderer,
         originalConfig,
         originalProcessSectionDelay,
         tmpDiv;
 
-    MathJax.Hub.Queue(
-    function() {
-        originalConfig = Lib.extendDeepAll({}, MathJax.Hub.config);
+    var setConfig = function() {
+        if(MathJaxVersion < 3) {
+            originalConfig = Lib.extendDeepAll({}, MathJax.Hub.config);
+
+            originalProcessSectionDelay = MathJax.Hub.processSectionDelay;
+            if(MathJax.Hub.processSectionDelay !== undefined) {
+                // MathJax 2.5+ but not 3+
+                MathJax.Hub.processSectionDelay = 0;
+            }
 
-        originalProcessSectionDelay = MathJax.Hub.processSectionDelay;
-        if(MathJax.Hub.processSectionDelay !== undefined) {
-            // MathJax 2.5+
-            MathJax.Hub.processSectionDelay = 0;
+            return MathJax.Hub.Config({
+                messageStyle: 'none',
+                tex2jax: {
+                    inlineMath: inlineMath
+                },
+                displayAlign: 'left',
+            });
+        } else {
+            originalConfig = Lib.extendDeepAll({}, MathJax.config);
+
+            if(!MathJax.config.tex.inlineMath) {
+                MathJax.config.tex.inlineMath = inlineMath;
```

Comment:
So if `inlineMath` is already set some other way we would honor that? Seems risky, means the graph would behave How about we always use our own, but if at some point we want to allow other delimiters we make a layout attr to provide them within the graph?

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/6073/commits/1de176ca87c125a1704b76f4f56be18409f02a2b.

Reply:
With https://github.com/plotly/plotly.js/pull/6073/commits/d5e10a7de0ab3ad4f8c83f7745a8101abcce221d looks great 😄